plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/sessionconfig/CodeScanSessionConfig.kt

```diff
@@ -88,7 +88,7 @@ class CodeScanSessionConfig(
 
         // Fail fast if the selected file size is greater than the payload limit.
         if (selectedFile != null && selectedFile.length > getPayloadLimitInBytes()) {
-            fileTooLarge(getPresentablePayloadLimit())
```

Comment:
Delete the `getPresentablePayloadLimit` function if we have no use for it anymore.